src/gtk/menu.cpp

```diff
@@ -1036,60 +1041,53 @@ static wxString GetGtkHotKey( const wxMenuItem& item )
         int code = accel->GetKeyCode();
         switch ( code )
         {
-            case WXK_F1:
-            case WXK_F2:
-            case WXK_F3:
-            case WXK_F4:
-            case WXK_F5:
-            case WXK_F6:
-            case WXK_F7:
-            case WXK_F8:
-            case WXK_F9:
-            case WXK_F10:
-            case WXK_F11:
-            case WXK_F12:
-            case WXK_F13:
-            case WXK_F14:
-            case WXK_F15:
-            case WXK_F16:
-            case WXK_F17:
-            case WXK_F18:
-            case WXK_F19:
-            case WXK_F20:
-            case WXK_F21:
-            case WXK_F22:
-            case WXK_F23:
-            case WXK_F24:
+            case WXK_F1:  case WXK_F2:  case WXK_F3:  case WXK_F4:  case WXK_F5:
+            case WXK_F6:  case WXK_F7:  case WXK_F8:  case WXK_F9:  case WXK_F10:
+            case WXK_F11: case WXK_F12: case WXK_F13: case WXK_F14: case WXK_F15:
+            case WXK_F16: case WXK_F17: case WXK_F18: case WXK_F19: case WXK_F20:
+            case WXK_F21: case WXK_F22: case WXK_F23: case WXK_F24:
                 hotkey += wxString::Format(wxT("F%d"), code - WXK_F1 + 1);
                 break;
 
                 // TODO: we should use gdk_keyval_name() (a.k.a.
                 //       XKeysymToString) here as well as hardcoding the keysym
                 //       names this might be not portable
-           case WXK_INSERT:
+            case WXK_INSERT:
                 hotkey << wxT("Insert" );
                 break;
             case WXK_DELETE:
                 hotkey << wxT("Delete" );
                 break;
             case WXK_UP:
-                hotkey << wxT("Up" );
+                if( flags )
```

Comment:
Nitpick (here and below):
```suggestion
                if ( flags )
```